src/index.html

```diff
@@ -10,8 +10,175 @@
+      method="post"
+      name="info-form"
+    >
+      <fieldset class="field-block">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="lname"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="fname"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number" name="number"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="bday"
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="checkbox"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="field-block">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label>
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="field-block">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          <label>
+           Do you love cats?
+            <input
+              type="radio"
+              name="cats"
+              value="Yes"
+            >Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="No"
+            >No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            value="#000"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="sleep-time"
+            step="2"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+            name="car"
+            multiple
+            required
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label>
+          How do you rate our work?
+          <input
+            type="range"
+            name="rate-work"
+            value="0"
+            min="0" max="10"
```

Comment:
you can't move some of the attributes to a new line, each of them should have a separate one
```suggestion
            type="range"
            name="rate-work"
            value="0"
            min="0" 
            max="10"
```